app/src/main/java/org/standardnotes/notes/frag/NoteFragment.kt

```diff
@@ -56,16 +54,13 @@ class NoteFragment : Fragment(), SyncManager.SyncListener {
 
     override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        val context = activity as AppCompatActivity
-        context.setSupportActionBar(toolbar)
-        context.supportActionBar?.setDisplayShowHomeEnabled(true)
-        context.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        activity.setTitle(R.string.title_activity_note)
 
         val noteUuid =
                 savedInstanceState?.getString(NoteListFragment.EXTRA_NOTE_ID) ?:
                         arguments?.getString(NoteListFragment.EXTRA_NOTE_ID)
-        if (noteUuid != null) {
-            note = SApplication.instance.noteStore.getNote(noteUuid)!!
+        if (noteUuid != "") {
+            note = SApplication.instance.noteStore.getNote(noteUuid!!)!!
```

Comment:
See above, which is why you now need the !! I-love-nulls command. Kotlin is clever!